jekyll/_cci2/pipelines.md

```diff
@@ -26,6 +26,15 @@ The output of each job can be opened in a new tab (in either raw or formatted st
 
 ![Download and share job step output](/docs/assets/img/docs/pipelines-job-output.png)
 
+## VS Code extension
```

Comment:
Once this PR is done, let't maybe look at converting all these pages to adoc and making this part a snippet?